src/stretchy.js

```diff
@@ -0,0 +1,179 @@
+    xrightharpoonup : [0.261, 0.261, "xrightharpoon"],
+    xhookleftarrow : [0.261, 0.261, "hookleftarrow"],
+    xhookrightarrow : [0.261, 0.261, "hookrightarrow"],
+    overlinesegment : [0.334, 0, "linesegment"],
+    underlinesegment : [0.334, 0, "linesegment"],
+    xmapsto : [0.261, 0.261, "mapsto"],
+    xrightharpoondown : [0.261, 0.261, "xrightharpoondown"],
+    xleftharpoondown : [0.261, 0.261, "xleftharpoondown"],
+    xrightleftharpoons : [0.358, 0.358, "rightleftharpoons"],
+    xleftrightharpoons : [0.358, 0.358, "leftrightharpoons"],
+    overgroup : [0.262, 0, "overgroup"],
+    undergroup : [0.262, 0, "undergroup"],
+    xtwoheadleftarrow : [0.167, 0.167, "twoheadleftarrow"],
+    xtwoheadrightarrow : [0.167, 0.167, "twoheadrightarrow"],
+    xLongequal : [0.167, 0.167, "longequal"],
+    xtofrom : [0.264, 0.264, "tofrom"],
+};
+
+const svgSpan = function(group, options) {
+    // Create a span with class(es) that refer to the background-image
+    // and/or the mask-image.
+    const label = group.value.label.substr(1);
+    let height = 0;
+    let depth = 0;
+    const classArray = ["stretchy"];
+    let fileName = "";
+
+    if (utils.contains(["widehat", "widetilde", "undertilde"], label)) {
+        // There are four SVG images available for each function.
+        // Choose a taller image when there are more characters.
+        const numChars = group.value.value.length;
+        if (numChars > 5) {
+            height = 0.34;
+            fileName = (label === "widehat" ? "widehat" : "tilde") + "4";
+        } else {
+            const imgIndex = [1, 1, 2, 2, 3, 3][numChars];
+            if (label === "widehat") {
+                height = [0, 0.16, 0.23, 0.23, 0.28, 0.28][numChars];
+                fileName = "widehat" + imgIndex;
+            } else {
+                height = [0, 0.15, 0.195, 0.195, 0.26, 0.26][numChars];
+                fileName = "tilde" + imgIndex;
+            }
+        }
+    } else {
+        const imgData = katexImagesData[label];
+        height = imgData[0];
+        depth = imgData[1];
+        fileName = imgData[2];
+        if (label.substr(0, 1) === "x") {
+            classArray.push("x-arrow");     // Lengthen the arrow via padding.
+        }
+    }
+
+    let node;
+    if (utils.contains(["overbracket", "underbracket"], label)) {
+        node = buildCommon.makeSpan(["stretchy", label], [], options);
+        // Deal with the optional arguments.
+        if (group.value.thickness) {
+            const lineWt = group.value.thickness.value + "em";
+            node.style.borderWidth = (group.value.isOver ? lineWt : 0) + " " +
+                lineWt + " " + (group.value.isOver ? 0 : lineWt) + " " + lineWt;
+        }
+        if (group.value.height) {
+            height = group.value.height;
+            node.style.height = height + "em";
+        }
+
+    } else if (options.color) {
+        classArray.push(fileName);         // Set span height and IE image.
```

Comment:
Very much a live comment.

This class sets a `background-image` and sets the span height. In the next two line of code, I apply two more classes, which CSS will apply only to browsers that support CSS `mask`.  

Because IE and Edge don't support `mask`, they will render the fall back solution defined in the first class, a black `background-image`.  And the CSS for span height, defined by the first class, remains valid for all browsers.